mmpose/core/evaluation/top_down_eval.py

```diff
@@ -313,9 +314,13 @@ def keypoints_from_heatmaps(heatmaps,
         scale (np.ndarray[N, 2]): Scale of the bounding box
             wrt height/width.
         post_process (bool): Option to use post processing or not.
-        unbiased (bool): Option to use unbiased decoding.
+        unbiased (bool): Option to use unbiased decoding. Mutually
+            exclusive with megvii.
             Paper ref: Zhang et al. Distribution-Aware Coordinate
             Representation for Human Pose Estimation (CVPR 2020).
+        megvii (bool):  Option to use megvii's way of post processing.
```

Comment:
for mutually exclusive options, could they be merged into a selective option, something like `post_process=None|unbiased|megvii`?

Reply:
Yes, but if we do it this way, we need to change all the config files.

Reply:
We can change them in a backward-compatible way. We add the new way of `post_process` option while keep the old way functional, and emit an deprecation warning for the old ways.

Reply:
With detailed and friendly migration message, I believe it will be a smooth transition